modules/images-configuration-registry-mirror.adoc

```diff
@@ -142,8 +142,8 @@ $ oc debug node/ip-10-0-147-35.ec2.internal
 Starting pod/ip-10-0-147-35ec2internal-debug ...
 To use host binaries, run `chroot /host`
 
-sh-4.3# chroot /host
-sh-4.3# cat /etc/containers/registries
+sh-4.4# chroot /host
```

Comment:
I think the changes in this file need to be reverted. `sh-4.3` (or 4.2, or 4.4 for that matter) are shell prompts and don't have anything to do with the OCP version.

Reply:
See https://github.com/openshift/openshift-docs/pull/21456/files#r414132734